### ISSUE TITLE
Media Cards: 400% zoom does not preserve content - 1.4.10 Reflow 

### ISSUE THREAD

**Issue by syarwick** [stephenravenscraft/loyola-template-4.0#78]
At 400% zoom, when the "Default Card Display" is not checked, the linked title and readout text overflow where the text cannot be read.

**Comment by syarwick23** [comment]
Fixed
